.circleci/config.yml

```diff
@@ -32,13 +32,53 @@ jobs:
           name: Authenticate with registry
           command: |
             echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/.npmrc
-            echo "//registry.yarnpkg.com/:_authToken=$NPM_TOKEN" >> ~/.npmrc 
+            echo "//registry.yarnpkg.com/:_authToken=$NPM_TOKEN" >> ~/.npmrc
       - run:
           name: Publish @pxblue/publish
           command: |
             cd pxb-publish
             bash scripts/publish.sh -b $CIRCLE_BRANCH
 
+  publish_pxblue_tag:
+    docker:
+      - image: circleci/node:12.16.3-browsers
+    steps:
+      - checkout
+      - run:
+          name: Authenticate with registry
+          command: |
+            echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/.npmrc
+            echo "//registry.yarnpkg.com/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+      - run:
+          name: Publish @pxblue/tag
+          command: |
+            cd pxb-tag
+            bash ../pxb-publish/scripts/publish.sh -b $CIRCLE_BRANCH
+
+  tag_pxblue_publish:
+    docker:
+      - image: circleci/node:12.16.3-browsers
+    steps:
+      - checkout
+      - run:
+          name: Tag @pxblue/publish
+          command: |
+            cd pxb-publish
+            yarn add @pxblue/tag
```

Comment:
Feels kind of weird to install the package like this...what about just making it a dev dependency?

Reply:
We can add it as a dev-dependency instead to clean up the config.yml. The reason why it's here is because this command will only run in a circle CI env so I didn't see much benefit to install the dependency outside of linux envs.

Reply:
You could in theory want to create a release tag locally, similarly to how we sometimes need to publish a package locally

Reply:
This command won't work on a mac though because the github CLI command only works for a linux/ubuntu environment.  

If we didn't need to install the Github CLI for this thing to work then we could get it to work on all platforms.

Alternative approach would be for us to tell users to install github cli locally, we run the `gh` command, if it errors the script installs github cli.